index.md

```diff
@@ -9,6 +9,14 @@ Paragraph of introductory material.
 > A short paragraph describing what learners need to know
 > before tackling this lesson.
 
+> ## Getting ready {.getready}
+>
+> Download [this file](https://github.com/swcarpentry/lesson-example/archive/gh-pages.zip)
```

Comment:
I'd prefer not to include the link here, because I don't want them to download the whole repo, but only the data files. This is because learners will have trouble navigating the repo files to find the data files.

I'd write:

"A short paragraph describing what learners need to do before tackling this lesson" 

So it's parallel to the prerequisites above.